src/category_theory/monoidal/of_has_finite_products.lean

```diff
@@ -208,6 +208,28 @@ def monoidal_of_has_finite_products [has_terminal C] [has_binary_products C] : m
   associator_naturality' := @prod.associator_naturality _ _ _, }
 end
 
+section
+local attribute [instance] monoidal_of_has_finite_products
+
+open monoidal_category
+
+/--
+The monoidal structure coming from finite products is symmetric.
+-/
+@[simps]
+def symmetric_of_has_finite_products [has_terminal C] [has_binary_products C] :
+  symmetric_category C :=
+{ braiding := limits.prod.braiding,
+  braiding_naturality' := λ X X' Y Y' f g,
+    by { dsimp [tensor_hom], ext, simp, simp, }, -- haven't seen that before!
```

Comment:
This clearly looks like an issue that should be investigated!

Reply:
This one has me bamboozled. Note that `dsimp, simp` doesn't work here, the first `dsimp` just fails, so this is a different issue than discussed above.

Reply:
I'm an idiot. `ext` produced two goals.